Viewer/src/managers/sceneManager.ts

```diff
@@ -798,6 +809,12 @@ export class SceneManager {
                 });
             });
         }
+        this._vrHelper.onEnteringVRObservable.add(() => {

```

Comment:
are those observables working now aas expected?

Reply:
With my proposed changes exitVR() should only be executing its logic once when the user exits VR (as compared to multiple times before). So these observables should only be fired once per entry/exit.